auxx/src/Command/DumpBlockchain.hs

```diff
@@ -39,84 +43,171 @@ import           Mode (MonadAuxxMode)
 -- Each epoch will be at <outFolder>/epoch<epochIndex>.cbor.
 dumpBlockchain
     :: (MonadAuxxMode m, MonadIO m)
-    => FilePath
+    => Maybe EpochIndex                 -- ^ Start epoch (Nothing = 0)
```

Comment:
It would probably be better to comment that both ends of the range are inclusive.